Web Inspector: align runtimes in the audit tests

francescorn opened the pull request:
#### 7b4e559c1d8a3eacedee53777435ca375b6a3635
<pre>
Web Inspector: align runtimes in the audit tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=268257">https://bugs.webkit.org/show_bug.cgi?id=268257</a>

Reviewed by Devin Rousso.

Change text-align:center to text-align:right and remove display:inline-block to align the runtimes
in .content-view.audit-test-case &gt; header &gt; .metadata &gt; .duration in AuditTestCaseContentView.css.

* Source/WebInspectorUI/UserInterface/Views/AuditTestCaseContentView.css:
(.content-view.audit-test-case &gt; header &gt; .metadata &gt; .duration):

Canonical link: <a href="https://commits.webkit.org/275298@main">https://commits.webkit.org/275298@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/52cade33041baa2c348e2fd066b5529836b9cdc1

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/41468 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/20482 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/43846 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/44035 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/37560 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/23588 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/17812 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/34283 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/42042 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/17409 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/35710 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/14943 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/15125 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/36714 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/45390 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/37655 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/37034 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/40783 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/16283 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/13360 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/39188 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/17902 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9292 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/17957 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/17546 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

dcrousso reviewed (changes requested):
personally i like the look of it better when center aligned as these values aren't necessarily meant to be compared (which is a benefit of right alignment with numbers, i.e. "which of these values is smallest/largest").  i think it also breaks the "grid" we have where the result level `WI.ScopeBar` and run date being aligned right and the pass percentage and run time being center aligned.  i dunno.  it just feels a bit weird to me.

can you explain the motivation for this change and/or why you feel the new UI is better?

dcrousso reviewed (approved):
i tried living with this for a bit and i think ive changed my mind and agree that it'd be better to align right

francescorn committed:
Web Inspector: align runtimes in the audit tests
https://bugs.webkit.org/show_bug.cgi?id=268257

Reviewed by Devin Rousso.

Change text-align:center to text-align:right and remove display:inline-block to align the runtimes
in .content-view.audit-test-case > header > .metadata > .duration in AuditTestCaseContentView.css.

* Source/WebInspectorUI/UserInterface/Views/AuditTestCaseContentView.css:
(.content-view.audit-test-case > header > .metadata > .duration):

Canonical link: https://commits.webkit.org/275298@main
